package.json

```diff
@@ -10,7 +10,7 @@
     "dev:node": "tsc -p tsconfig.node.json --watch",
     "build:client": "tsc -p tsconfig.client.json",
     "build:node": "tsc -p tsconfig.node.json",
-    "build:components": "mkdir -p lib/client/components && cp -r src/client/components lib/client",
+    "build:components": "cpx \"src/client/components/**/*\" lib/client/components",
```

Comment:
Maybe we can omit the `\"`?

Reply:
I think these `\"`s are required because some shells fail expanding glob expression. (ref: [1](https://www.npmjs.com/package/cpx#example), [2](https://medium.com/@jakubsynowiec/you-should-always-quote-your-globs-in-npm-scripts-621887a2a784))